### ISSUE TITLE
Required reason APIs

### ISSUE THREAD

**Issue by VlGalkin** [kean/Pulse#216]
Hi, a new Apple policy is coming in the fall, tell me, do you plan to add required reason APIs?

I saw that you have 2 API categories in Pulse
File timestamp and User Defaults.

**Comment by kean** [comment]
Yes, I'm planning to address it once it becomes mandatory. I hope Apple reverses course.

**Comment by VlGalkin** [comment]
Any update on this please?

**Comment by mbuchetics** [comment]
@kean  it is mandatory now ...

**Comment by kean** [comment]
The privacy manifest was added in https://github.com/kean/Pulse/releases/tag/4.1.0.
